### PR TITLE
Add missing Kirundi translations

### PR DESCRIPTION
**Pull request by Diama1**
#1291

### CODE REVIEW

File: client/src/elm/Translate.elm [hunk 1]
```diff
@@ -23496,13 +23496,13 @@ translationSet trans =
...
                     , kinyarwanda = Just "Umwana ahita ananirwa iyo atangiye kurya"
-                    , kirundi = Nothing
+                    , kirundi = Just "Uruyoya ruruha vuba igihe rwonka"
                     }
 
                 SymptomCoughingOrTearingWhileFeeding ->
                     { english = "Coughing/tearing while feeding (<6 months)"
                     , kinyarwanda = Just "Gukorora/Kwiriza iyo atangiye kurya (munsi y'amezi 6)"
-                    , kirundi = Nothing
+                    , kirundi = Just ""
```
Comment (anvmn) [inline client/src/elm/Translate.elm]: Why this change?

**Review by anvmn — COMMENTED** [review]
@Diama1 ?

File: client/src/elm/Translate.elm (hunk 1 above)
Comment (Diama1) [reply client/src/elm/Translate.elm]: I might have missed it I will revert that

Comment (anvmn) [reply client/src/elm/Translate.elm]: Reverted myself.
